test/functional/core/startup_spec.lua

```diff
@@ -1097,3 +1097,23 @@ describe('user session', function()
     eq(1, eval('g:lua_session'))
   end)
 end)
+
+describe('inccommand on ex mode', function()
+  it('should not preview', function()
+    clear()
+    local screen
+    screen = Screen.new(60, 10)
+    screen:attach()
+    local id = funcs.termopen({ nvim_prog, '-u', 'NONE', '-c', 'set termguicolors', '-E', 'test/README.md' })
```

Comment:
```suggestion
    local id = funcs.termopen({ nvim_prog, '--clean', '-c', 'set termguicolors', '-E', 'test/README.md' })
```

Reply:
seems like if add `--clean` also need set VIMRUNTIMPATH. because in main.c will source syntax.vim ..

```C
  if (!vimrc_none || params.clean) {
    // Sources filetype.lua unless the user explicitly disabled it with :filetype off.
    filetype_maybe_enable();
    // Sources syntax/syntax.vim. We do this *after* the user startup scripts so that users can
    // disable syntax highlighting with `:syntax off` if they wish.
    syn_maybe_enable();
  }
```

Reply:
Regardless of `VIMRUNTIMEPATH`, we should do either `--clean` or `-u NONE -i NONE`, shouldn't we?